<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLogin</class>
 <widget class="QDialog" name="DlgLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>664</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login Window</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>8</number>
   </property>
   <property name="styleSheet">
    <string notr="true"> #groupBox {
                                border: 3px solid gray;
                                border-radius: 40px;
                                background: white;
                                background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
                            }
                            
                            QPushButton {
                                color: white;
                                background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
                                border-width: 1px;
                                border-color: #339;
                                border-style: solid;
                                border-radius: 7;
                                padding: 3px;
                                font-size: 10px;
                                padding-left: 5px;
                                padding-right: 5px;
                                min-width: 50px;
                                max-width: 50px;
                                min-height: 13px;
                                max-height: 13px;
                            }                  </string>
   </property>
   <property name="title">
    <string>User Login</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User Account</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAccount">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOk">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="icon">
     <iconset resource="../resource/icons.qrc">
      <normaloff>:/button/images/button/ok.png</normaloff>:/button/images/button/ok.png</iconset>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="icon">
     <iconset resource="../resource/icons.qrc">
      <normaloff>:/button/images/button/quit.gif</normaloff>:/button/images/button/quit.gif</iconset>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSetting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../resource/icons.qrc">
      <normaloff>:/button/images/button/setting.png</normaloff>:/button/images/button/setting.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Language</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbLanguage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>简体中文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="leMessage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
